ocaml/xapi/repository.ml

```diff
@@ -727,20 +727,23 @@ let apply_updates' ~__context ~host ~updates_info ~livepatches ~acc_rpm_updates
     =
   (* This function runs on coordinator host *)
   let open Guidance in
-  let guidance =
+  let guidance' =
     reduce_guidance ~updates_info ~updates:acc_rpm_updates ~livepatches
-    (* EvacuateHost should be carried out before host.apply_updates *)
-    |> List.map (fun (k, l) -> (k, List.filter (fun x -> x <> EvacuateHost) l))
   in
   let mandatory =
-    match List.assoc_opt Mandatory guidance with
+    match List.assoc_opt Mandatory guidance' with
     | Some tasks ->
         tasks
     | None ->
         warn "No mandatory guidance found. Ignore it." ;
```

Comment:
Does this requires a warning? Or debug?

Reply:
Mandatory guidance is expected to be there. But the absence of it should not block the update.